src/nvim/lua/executor.c

```diff
@@ -254,8 +254,8 @@ static int nlua_luv_thread_common_cfpcall(lua_State *lstate, int nargs, int nres
     if (status == LUA_ERRMEM && !(flags & LUVF_CALLBACK_NOEXIT)) {
       // Terminate this thread, as the main thread may be able to continue
       // execution.
-      os_errmsg(e_outofmem);
-      os_errmsg("\n");
+      fprintf(stderr, "%s", e_outofmem);
```

Comment:
we can clean up all callsites with static strings like these, just `fprintf(stderr, e_outofmem)` would work. Although right here we might want `fprintf(stderr, "%s\n",  e_outofmem)` .  (imagine being able to actually use printf when writing to stdio lol)